### PR TITLE
Update to the main page

### PR DESCRIPTION
**Pull request by TechPunk316**
Added the following:
- page weight "1" so it appears at the top of the sidebar
- restructured sentence to avoid starting with lowercase

### CODE REVIEW

File: source/documentation/index.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+title: Government Security Group standards and guidance 
+weight: 1
+---
+
 # Government Security Group standards and guidance 
 
 The Government Security Group (GSG) is responsible for security across the UK government. 
 
-security.gov.uk will provide up-to-date guidance and standards to support security professionals across government.
+Government security professional can find up-to-date guidance and standards on security.gov.uk.
```
Comment (jonodrew) [inline source/documentation/index.md:10]: professionals?

**Review by jonodrew — CHANGES_REQUESTED** [review]
once spelling change, otherwise looks good

File: source/documentation/index.md (hunk 1 above)
Comment (TechPunk316) [reply source/documentation/index.md:10]: Good spot will update.

**Review by jonodrew — APPROVED** [review]

**Comment by jonodrew** [comment]
LGTM :)

**Pull request merged**